docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+#### Configure the `orderer.yaml` file for each orderer
+
+Follow the instructions in the [ordering service deployment guide](../deployorderer/ordererdeploy.html) to build an ordering service with three ordering nodes. However, because the system channel is no longer required when you start an ordering service, you can skip the entire process of [generating the genesis block](../deployorderer/ordererdeploy.html#creating-the-genesis-block) in those instructions. In addition, when you configure the `orderer.yaml` file for each orderer, there are a few other modifications you need to make to leverage this feature. You can refer to the orderer [sampleconfig](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/orderer.yaml) for more information about these parameters.
+
+- `General.BootstrapMethod` - Set this value to `none`. Because the system channel is no longer required, the `orderer.yaml` file on each orderer needs to be configured with `BootstrapMethod: none` which means that no bootstrap block is required or used to start up the ordering service.
+- `Admin.ListenAddress` - The orderer admin server address (host and port) that can be used by the `osnadmin` command to configure channels on the ordering service. This value should be a unique `host:port` combination to avoid conflicts.
+- `Admin.TLS.Enabled:` - This value must be set to `true`.
+- `Admin.TLS.PrivateKey:` - The path to and file name of the orderer private key issued by the TLS CA.
+- `Admin.TLS.Certificate:` - The path to and file name of the orderer signed certificate issued by the TLS CA.
+- `Admin.TLS.RootCAs:`  - The path to and file name of the orderer TLS CA Root certificate.
+- `Admin.TLS.ClientAuthRequired:` This value must be set to `true`. Note that while mutual TLS is required for all operations on the orderer Admin endpoint, the entire network is not required to use Mutual TLS.
+- `Admin.TLS.ClientRootCAs:` - The path to and file name of the admin client TLS CA Root certificate. In the folder structure above, this is `admin-client/client-tls-ca-cert.pem`.
+- `ChannelParticipation.Enabled:` - Set this value to `true` to enable this feature on the orderer.
+
+**Start each orderer**  
+
+1. If you have not already, set the path to the location of the Fabric binaries on your system:
+    ```
+    export PATH=<path to download location>/bin:$PATH
+    ```
+2. In your terminal window set the `FABRIC_CFG_PATH` to point to the location of the `orderer.yaml` file, relative to where you are running the Fabric commands from. For example, if you download the binaries, and run the commands from the `/bin` directory and the `orderer.yaml` is under `/config`, the path would be:
+    ```
+    export FABRIC_CFG_PATH=../config
+    ```
+3. You can now start the ordering service by running the following command on each ordering node:
+    ```
+    orderer start
+    ```
+
+When the ordering service starts successfully, you should see something similar to the following output:
```

Comment:
Again, I wouldn't think of this as an "ordering service" starting yet. At this point it is just three independent OSNs starting and each displaying the same output.